discovery-kubernetes-api/src/main/mima-filters/1.6.0.backwards.excludes

```diff
@@ -0,0 +1,11 @@
+ProblemFilters.exclude[DirectAbstractMethodProblem]("akka.discovery.ServiceDiscovery.lookup")
```

Comment:
Not actually sure why this one fires... tbh: the method is still there (in `BaseKubernetesApiServiceDiscovery`)?

Reply:
Also, that class is not in this project so quite unexpected.